vector/src/main/java/im/vector/app/features/invite/InviteUsersToRoomViewModel.kt

```diff
@@ -87,6 +87,9 @@ class InviteUsersToRoomViewModel @AssistedInject constructor(
                         }
                         _viewEvents.post(InviteUsersToRoomViewEvents.Success(successMessage))
                     }
+                    .catch { cause ->
+                        _viewEvents.post(InviteUsersToRoomViewEvents.Failure(cause))
+                    }.collect()
```

Comment:
The other change in #6107 was to replace this by `.launchIn(viewModelScope)` and remove the first line `viewModelScope.launch {`. Did you see it? Maybe it was not working as expected?

Reply:
I reverted that while trying some alternatives using `.collect()` and I forgot to set back the `.launchIn(...)` call. It should be fixed now.